provers/blevm/blevm-prover/src/rsp.rs

```diff
@@ -0,0 +1,50 @@
+use alloy_provider::ProviderBuilder;
+use eyre::Context;
+use reth_chainspec::ChainSpec;
+use rsp_host_executor::EthHostExecutor;
+use rsp_primitives::genesis::Genesis;
+use rsp_rpc_db::RpcDb;
+use std::sync::Arc;
+
```

Comment:
we should try add file descriptions at the top of the files in the src to make it easier to understand at first glance what purpose they serve without having to delve into the code

Reply:
Will do.